sds_data_manager/lambda_code/IAlirtCode/ialirt_pointing_schedule.py

```diff
@@ -0,0 +1,89 @@
+"""Lambda to generate and upload antenna pointing schedule files to s3."""
+
+import json
+import logging
+import os
+from datetime import datetime, timedelta, timezone
+
+import boto3
+import imap_processing.ialirt.constants
+
+from sds_data_manager.lambda_code.IAlirtCode.ialirt_coverage import (
+    get_latest_spice_kernels,
+    setup_spice_file,
+)
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+KERNELS = {
```

Comment:
Does this need to be a set, or can it be a list of kernels? Where does it get used?
```suggestion
KERNELS = [
```